aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -121,6 +121,41 @@ builder.On<string, string>("ReceiveMessage", (user, message) => ...
 await builder.StartAsync();
 ```
 
+### SignalR seamless reconnect
+
+SignalR seamless reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
+
+Seamless reconnect achieves this by:
+
+* Temporarily buffering data on the server and client.
+* Acknowledging messages received (ACK-ing) by both the server and client.
+* Recognizing when a connection is returning and replaying messages that may have been sent while the connection was down.
+
+To opt-in to seamless reconnect, update your .NET client code to enable the `UseAcks` option:
```

Comment:
Since preview6 is already out, you need to additionally mention that the server **must opt-in** via code like
```csharp
app.MapHub<AppHub>("/default", o => o.AllowAcks = true);
```